common/src/main/java/me/confuser/banmanager/common/commands/CommandParser.java

```diff
@@ -75,6 +78,7 @@ public Reason getReason(int start) {
       String replace = plugin.getReasonsConfig().getReason(key);
 
       if (replace != null) reason = reason.replace("#" + key, replace);
+      else if(plugin.getConfig().isBlockInvalidReasons()) invalidReason = true;
```

Comment:
Can this be wrapped in braces instead? I.e.

```suggestion
      if (replace != null) {
        reason = reason.replace("#" + key, replace);
      } else if(plugin.getConfig().isBlockInvalidReasons()) {
        invalidReason = true;
      }
```